core/os/device/os.go

```diff
@@ -41,3 +41,7 @@ func (o1 *OS) CompareVersions(o2 *OS) VersionMatch {
 		return CompleteMatch
 	}
 }
+
+func (o *OSKind) Choose(c interface{}) {
```

Comment:
Is this used anywhere?

Reply:
Yup, bu the flag parsing. Don't you love go's implicit interfaces?!